contracts/externals/ens/profiles/TextResolver.sol

```diff
@@ -34,4 +34,4 @@ contract TextResolver is ResolverBase {
     function supportsInterface(bytes4 interfaceID) public pure returns(bool) {
         return interfaceID == TEXT_INTERFACE_ID || super.supportsInterface(interfaceID);
     }
-}
+}
```

Comment:
Some files are missing new lines at the end

Reply:
we should add them in ... what does our linter think about this ?

Reply:
I think that the formating tool does not check the external files, maybe we should add them. To me it is irrelevant and not so important.

Reply:
I explicitly ignored them, didn't want to format external code. But you can add them to format.sh if you think it's a good idea